release-notes/docs/3.5.0.md

```diff
@@ -0,0 +1,179 @@
+# 3.5.0 Release Notes
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [Performance](#performance)
+- [Sort muted contacts](#sort-muted-contacts)
+- [SIH integration](#sih-integration)
+- [And more...](#and-more)
+
+## Known issues
+
+None.
```

Comment:
might link the performance regression?

Reply:
Good call. That came in after I wrote this...